Libraries/src/Amazon.Lambda.AspNetCoreServer/Internal/HostingOptions.cs

```diff
@@ -0,0 +1,8 @@
+﻿using Amazon.Lambda.Core;
+
+namespace Amazon.Lambda.AspNetCoreServer.Internal;
```

Comment:
Since `HostingOptions` is used in the public method `AddAWSLambdaHosting` the type should not be in the `Internal` namespace and needs node.

Reply:
Done.